scripts/paper_plots/eval_sample_factory.py

```diff
@@ -34,7 +33,7 @@
 from sample_factory.utils.utils import log, AttrDict
 from examples.sample_factory_files.run_sample_factory import register_custom_components
 
-from cfgs.config import PROCESSED_VALID_NO_TL, PROCESSED_TRAIN_NO_TL, ERR_VAL
+from cfgs.config import PROCESSED_VALID_NO_TL, PROCESSED_TRAIN_NO_TL, ERR_VAL, set_display_window
```

Comment:
It seems this line is over 80 characters. Please make it as
```
from cfgs.config import (PROCESSED_VALID_NO_TL, PROCESSED_TRAIN_NO_TL, ERR_VAL, set_display_window)
```
And then run yapf to format it.